File: Cs/Solution/AlternatingSum.cs
using System;

namespace Solution
{
	public partial class Solution
	{
		public int AlternatingSum(int[] nums)
		{
			int sum = 0;
			for (int i = 0; i < nums.Length; i += 2)
			{
				sum += nums[i];
			}
			for (int i = 1; i < nums.Length; i += 2)
			{
				sum -= nums[i];
			}
			return sum;
		}
	}
}

File: Rust/src/solution/alternating_sum.rs
use super::Solution;

impl Solution {
    pub fn alternating_sum(nums: Vec<i32>) -> i32 {
        let mut sum = 0;
        nums.iter().enumerate().for_each(|(i, &num)| {
            sum += if i % 2 == 0 { num } else { -num };
        });
        sum
    }
}
